### DISCUSSION TITLE
Le Petit Prince sub-EN100, post-fMRIPrep denoised data look odd

### DISCUSSION THREAD

**Discussion by tsalo** [ME-ICA/tedana#899] [Q&A] [answered]
The preprocessed data are available at https://gin.g-node.org/ME-ICA/ds003643-fmriprep-derivatives.
The raw data are available at https://openneuro.org/datasets/ds003643/versions/2.0.1.
The code used to organize and preprocess the data with fMRIPrep v21.0.0 are available at https://github.com/ME-ICA/ds003643.

Here is the tedana command I used:

```bash
run_number=1

tedana \
    -d /Users/taylor/Documents/datasets/ds003643/derivatives/fmriprep/sub-EN100/func/sub-EN100_task-lppEN_run-${run_number}_echo-1_desc-preproc_bold.nii.gz \
    /Users/taylor/Documents/datasets/ds003643/derivatives/fmriprep/sub-EN100/func/sub-EN100_task-lppEN_run-${run_number}_echo-2_desc-preproc_bold.nii.gz \
    /Users/taylor/Documents/datasets/ds003643/derivatives/fmriprep/sub-EN100/func/sub-EN100_task-lppEN_run-${run_number}_echo-3_desc-preproc_bold.nii.gz \
    -e 12.8 27.5 43 \
    --mask /Users/taylor/Documents/datasets/ds003643/derivatives/fmriprep/sub-EN100/func/sub-EN100_task-lppEN_run-${run_number}_desc-brain_mask.nii.gz \
    --out-dir /Users/taylor/Documents/datasets/ds003643/derivatives/tedana/sub-EN100/func/sub-EN100_task-lppEN_run-${run_number} \
    --prefix sub-EN100_task-lppEN_run-${run_number} \
    --debug \
    --fittype loglin \
    --tedort
```

The denoised data carpet plot looks pretty bad (see below), and the accepted components only explain ~3% of the variance.

![image](https://user-images.githubusercontent.com/8228902/202540413-5f75edbe-ba2c-45e1-bf2b-b1b5f4a9e3d0.png)

I've attached the HTML report, metrics files, and figures: [report_results.zip](https://github.com/ME-ICA/tedana/files/10034901/report_results.zip)

@handwerkerd @jbteves WDYT?

**Comment by handwerkerd** [comment, marked as answer]
The highest variance components have a very large spike at the beginning of the scans. That is usually a sign that the non-steady state volumes weren't removed. Siemens removes those by default, but GE keeps them. Looking at the openneuro description of these data, some participants were acquired on a Siemens scanner and others on a GE scanner. Any chance this run was GE scanner without steady state volumes removed?

**Reply by tsalo** [reply]
I noticed the spikes, but I (incorrectly) assumed that wouldn't be a problem for the ICA. You're right- the English-language participants in Le Petit Prince were scanned with a GE Discovery MR750 at Cornell. I'll try dropping the non-steady-state volumes flagged by fMRIPrep before running tedana.

**Reply by tsalo** [reply]
I ended up dropping the non-steady-state volumes (although I had to drop more than the volumes flagged by fMRIPrep, unfortunately). The denoised data look _much_ better (see carpet plot below), though the accepted components still only explain about 7% of the variance.

![image](https://user-images.githubusercontent.com/8228902/202776850-c9cb3c13-b24c-4a27-9f14-5238b2fcd873.png)

I've attached the new HTML report, metrics files, and figures: [report_results_2.zip](https://github.com/ME-ICA/tedana/files/10043920/report_results_2.zip)

WDYT? Is 7% reasonable?

**Reply by handwerkerd** [reply]
The total number of components and the kappa vs rho values of the accepted components look reasonable to me. What I do see is that you have a few rejected components with massively high variance. A component that is clearly linear drift explains 42.25% of the variance and two more drift components are another 22.04% and 5.43%. The more noise there is to remove from the data the less remaining variance will be accepted. Nothing about component selection with tedana looks obviously wrong to me.

The question is why these data have so much linear drift. A participant who slowly moves their head out of the head coil can cause something like this & you'd be able to see that in motion parameters. Otherwise look to see if there's a similar magnitude of drift in other runs.

**Reply by tsalo** [reply]
Thanks!

I'll take a look at the other runs to see if they show similar levels of drift.

**Reply by tsalo** [reply]
Some do show massive drift components, but others don't. However, the ones that don't seem to have big spikes in their largest components, so I'm guessing that the participants made some large movements in those runs.
